src/apps/replication/client_lib/replication_common.h

```diff
@@ -33,16 +33,6 @@ using namespace ::dsn::service;
 
 namespace dsn { namespace replication {
 
-inline bool operator < (const global_partition_id& l, const global_partition_id& r)
```

Comment:
As commented above, this is explicitly put into this file to avoid code overwritten by code generation.